tests/unit/other.tcl

```diff
@@ -568,7 +568,8 @@ if {$::verbose} {
 set tempFileId [open $tempFileName w]
 set group [dict get [file attributes $tempFileName] -group]
 if {$group != ""} {
-    start_server [list tags {"repl external:skip"} overrides [list unixsocketgroup $group unixsocketperm 744]] {
+    set escaped_group "\"[string map {"\\" "\\\\"} $group]\""
+    start_server [list tags {"repl external:skip"} overrides [list unixsocketgroup $escaped_group unixsocketperm 744]] {
```

Comment:
Wouldn't it be more correct to do the escaping inside `start_server` so it works everywhere and for all options?

(`proc start_server` is defined in tests/support/server.tcl.)

Reply:
Probably yes. Do we have a list of all string options that we should escape?
Could we try to escape any string option if we don't have such list?

Reply:
I don't know. :) I think some investigation is needed to find out the best solution here. Now it's already merged so let's not worry too much about it.